src/algebra/CommRing/basic.lean

```diff
@@ -40,7 +40,7 @@ namespace CommRing
 
 instance (x : CommRing) : comm_ring x := x.str
 
-@[reducible] def is_comm_ring_hom {α β} [comm_ring α] [comm_ring β] (f : α → β) : Prop :=
+abbreviation is_comm_ring_hom {α β} [comm_ring α] [comm_ring β] (f : α → β) : Prop :=
```

Comment:
Is `abbreviation` documented somewhere? I couldn't find anything about the keyword in PIL or TPIL? What are the pros and cons?